templates/telco/osm/index.html

```diff
@@ -0,0 +1,473 @@
+      </p>
+    </div>
+    <hr class="p-separator" />
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h2>Flexible VIM options</h2>
+      <p>OSM supports all the major VIMs (virtualized infrastructure manager) in the ecosystem.</p>
+      <p>A single OSM instance can deploy network functions on OpenStack, AWS, VMware and k8s clusters.
+      </p>
+    </div>
+    <div class="col-6 u-align--center u-vertically-center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/b4525eba-Screenshot+from+2021-07-23+12-27-40.png",
+        alt="",
+        width="253",
+        height="121",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <hr class="p-separator" />
+  </div>
+  <div class="row">
+    <div class="col-6 u-align--center u-vertically-center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/9f21a863-OSM-5.svg",
+        alt="",
+        width="450",
+        height="134",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-6">
+      <h2>Enterprise support</h2>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Charmed OSM design and delivery</li>
+        <li class="p-list__item is-ticked">MANO stack enterprise support</li>
+        <li class="p-list__item is-ticked">Fully managed OSM</li>
+        <li class="p-list__item is-ticked">Onboarding network functions</li>
+      </ul>
+      <p>
+        <a href="/legal/ubuntu-advantage-service-description">Ubuntu Support&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h2>Participation in OSM Hackfests</h2>
+    <p>Being one of the pillars of the OSM project, Canonical regularly participates in the <a class="p-link--external" href="https://osm.etsi.org/wikipub/index.php/Past_OSM_Hackfests">events</a> organized by the OSM community and ETSI's Centre for Testing and Interoperability. For events like Hackfests and NFV Plugtests, Canonical also offers a partner cloud where multiple VNF vendors can deploy and test the interoperability of their Telco network functions.</p>
+    <p class="p-heading--four">52 Participants, More than 100 Network Functions onboarding</p>
+    <div class="u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/a1afcfd0-V.jpg",
+        alt="",
+        width="681",
+        height="409",
+        hi_def=True,
+        loading="lazy",
+        attrs={"class": "p-image--bordered"},
+        ) | safe
+      }}
+    </div>
+    <p>
+      <a href="https://charmed-osm.com/osm-hackfests">Discover our contributions&nbsp;&rsaquo;</a>
+    </p>
+  </div>
+</section>
+
+<section class="p-strip is-bordered is-deep">
+  <div class="u-fixed-width">
+    <h2>Interesting OSM Resources</h2>
```

Comment:
As discussed in our call just now - the `u-fixed-width` div should only contain the `h2`, and the `.row.p-divider` element should come up one indentation level to sit alongside the `u-fixed-width`, so that there isn't extra left and right padding on the divider block.